src/journal/journal-file.h

```diff
@@ -85,6 +85,7 @@ typedef struct JournalFile {
         bool compress_lz4:1;
         bool seal:1;
         bool defrag_on_close:1;
+        volatile bool archive:1;
```

Comment:
It's to prevent the compiler from caching the value from a previous iteration in journal_file_set_offline_internal().  It's not terribly performance-sensitive code, this loop blocks in fsyncs and uses atomics to progress through the various offlining states.